Tests/LinuxMain.swift

```diff
@@ -1,43 +1,10 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the SwiftNIO open source project
-//
-// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
-// Licensed under Apache License v2.0
-//
-// See LICENSE.txt for license information
-// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-//===----------------------------------------------------------------------===//
-//
-// LinuxMain.swift
-//
 import XCTest
 
-///
-/// NOTE: This file was generated by generate_linux_tests.rb
-///
-/// Do NOT edit this file directly as it will be regenerated automatically when needed.
-///
+import NIOExtrasTests
+import NIOHTTPCompressionTests
 
-#if os(Linux) || os(FreeBSD)
-   @testable import NIOExtrasTests
-   @testable import NIOHTTPCompressionTests
+var tests = [XCTestCaseEntry]()
+tests += NIOExtrasTests.__allTests()
+tests += NIOHTTPCompressionTests.__allTests()
 
-   XCTMain([
-         testCase(DebugInboundEventsHandlerTest.allTests),
-         testCase(DebugOutboundEventsHandlerTest.allTests),
-         testCase(FixedLengthFrameDecoderTest.allTests),
-         testCase(HTTPResponseCompressorTest.allTests),
-         testCase(JSONRPCFramingContentLengthHeaderDecoderTests.allTests),
-         testCase(JSONRPCFramingContentLengthHeaderEncoderTests.allTests),
-         testCase(LengthFieldBasedFrameDecoderTest.allTests),
-         testCase(LengthFieldPrependerTest.allTests),
-         testCase(LineBasedFrameDecoderTest.allTests),
-         testCase(QuiescingHelperTest.allTests),
-         testCase(RequestResponseHandlerTest.allTests),
-         testCase(WritePCAPHandlerTest.allTests),
-    ])
-#endif
+XCTMain(tests)
```

Comment:
What happened here?

Reply:
`swift test --generate-linuxmain`

Reply:
Why aren't we using the test generation script in this directory?

Reply:
habit, I guess :), re-generated
sidenote: don't we want to remove all those scripts since we can generate `LinuxMain` using SPM?

Reply:
It still re-generated a lot of files...

Reply:
> sidenote: don't we want to remove all those scripts since we can generate LinuxMain using SPM?

Nope, it's too easy to forget and doesn't work on Linux so it can't be enforced in CI. 

> It still re-generated a lot of files...

Delete the manifest file.

Reply:
Done

Reply:
It's a shame. Swift 5.1 can now (on Linux) also auto-discover the tests, but Swift 5.0 can not :(